### ISSUE TITLE
helm package templates hard coded rbac v1alpha1. Broken in k8s 1.6

### ISSUE THREAD

**Issue by peterrosell** [coreos/prometheus-operator#253]
Started a clean minikube with k8s version 1.6. kubectl version 1.6. helm version 2.2.0.
Installed tiller via helm init
Tried to install prometeus-operator with this command:
```helm install --name my-release opsgoodness/prometheus-operator```

Expected to see a successful installation of the operator, but got this error.
```
Error: release my-release failed: the API version in the data (rbac.authorization.k8s.io/v1alpha1) does not match the expected API version (rbac.authorization.k8s.io/v1beta1)
```
Running ```helm install --name my-release opsgoodness/prometheus-operator --set rbacEnable=false``` worked, but of course it didn't configure the rbac.

I saw in the templates that the version is hard coded. I think the best solution is to move the api version to a variable in the values.yaml file.

**Comment by brancz** [comment]
Interesting, `kubectl apply` automatically takes care of the conversion, but as we are targeting 1.6 it's fine to just change the version.

@mgoodness can you take care of this?

**Comment by mgoodness** [comment]
I'll have a fix shortly.

**Comment by mgoodness** [comment]
Fixed in #255. Note that the chart version has been reset to v0.0.1. You'll want to purge any previous deployments and `helm repo up` before installing.
